src/domain.py

```diff
@@ -130,21 +140,25 @@ def _get_parent_domain_of(domain):
         return _get_parent_domain_of(parent_domain)
 
 
-@is_unit_operation()
-def domain_add(operation_logger, domain, dyndns=False):
+@is_unit_operation(exclude=["subscribe"])
+def domain_add(operation_logger, domain, subscribe=None, no_subscribe=False):
     """
     Create a custom domain
 
     Keyword argument:
         domain -- Domain name to add
         dyndns -- Subscribe to DynDNS
-
+        subscribe -- Password used to later unsubscribe from DynDNS
+        no_unsubscribe -- If we want to just add the DynDNS domain to the list, without subscribing
```

Comment:
These option names are hella confusing x_x

If `subscribe` is the recovery password .. why not name it `recovery_password` ?

Reply:
Also the `no_(un)subscribe` is confusing ... the previous `dyndns` option was already handling that ... changing the name is going to cause boring backward compatibility issues (at least on the postinstall, I'm pretty sure some scripts have some `--ignore-dyndns` or similar options somewhere)